pizza_cutter/des_pizza_cutter/_coadd_slices.py

```diff
@@ -555,7 +555,11 @@ def _coadd_slice_inputs(
 
         image += (resampled_data['image'] * weight)
         noise += (resampled_data['noise'] * weight)
-        interp_se_frac += (resampled_data['interp_frac'] * weight)
+        # we force the interp fraction to be in [0, 1]
+        # because the lanczos interp doesn't do this natively
+        _mfrac = np.abs(resampled_data['interp_frac'])
```

Comment:
why take the absolute value first?  It seems to me that values below zero should end up zero

Reply:
The goal is to mark any pixel with interpolated flux. If lanczos is subtracting part of an interpolated value when remapping, we want to capture that. hence the absolute value.

Reply:
I see